client/src/docs-ui/ui-component-props/ui-component-props.types.tsx

```diff
@@ -0,0 +1,7 @@
+import {JsonDocsComponent, JSXBase} from "@stencil/core/internal";
+
+export type PropType = "attr" | "slots" | "css";
```

Comment:
This name may be too generic, can we look to give it a more detailed name?

Reply:
Changed to `WebComponentProps`!